sdk/program/src/pubkey.rs

```diff
@@ -54,6 +56,10 @@ impl From<u64> for PubkeyError {
     }
 }
 
+/// wasm-bindgen version of the Pubkey struct.
+/// This duplication is required until https://github.com/rustwasm/wasm-bindgen/issues/3671
+/// is fixed. This must not diverge from the regular non-wasm Pubkey struct.
+///
```

Comment:
Is this comment needed? There's only one version of this because it doesn't require #[wasm_bindgen(skip)]